Source/Core/DiscIO/DiscUtils.h

```diff
@@ -49,6 +49,24 @@ constexpr u32 WII_NONPARTITION_DISCHEADER_SIZE = 0x100;
 constexpr u32 WII_REGION_DATA_ADDRESS = 0x4E000;
 constexpr u32 WII_REGION_DATA_SIZE = 0x20;
 
+// Default block size for GCZ/WIA/RVZ images
+constexpr int PREFERRED_BLOCK_SIZE = 0x20000;  // 128 KiB
+
+// Because DVD timings are emulated as if we can't read less than an entire ECC block at once
+// (32 KiB - 0x8000), there is little reason to use a block size smaller than that.
+constexpr int PREFERRED_MIN_BLOCK_SIZE = 0x8000;  // 32 KiB
```

Comment:
"32 KiB" is mentioned twice, which is a little unnecessary.

(Also, I'm realizing now that the `-` looks confusingly much like a minus... But since it's from existing code, I'm not going to require you to fix it.)